apps/wpcom-block-editor/src/wpcom/features/tracking/site-editor-load.js

```diff
@@ -0,0 +1,15 @@
+import { select } from '@wordpress/data';
+import { registerPlugin } from '@wordpress/plugins';
+import tracksRecordEvent from './track-record-event';
+
+registerPlugin( 'wpcom-site-editor-load', {
+	render: () => {
+		// This also loads in the Post Editor context.
+		if ( ! select( 'core/edit-site' ) ) {
+			return;
+		}
+		// this is no longer, strictly speaking, a "calypso" page view, but this is for back compat post-un-iframing.
+		tracksRecordEvent( 'caplyso_page_view', { path: '/:post_type/:site' } );
```

Comment:
Minor typo here 🙂
```suggestion
		tracksRecordEvent( 'calypso_page_view', { path: '/:post_type/:site' } );
```

Reply:
aha oh no thanks for the catch @taipeicoder @autumnfjeld I'll fix it up and get this shipped today.